frontend/src/components/mine/Permit/MinePermitTable.js

```diff
@@ -0,0 +1,467 @@
+    key: "authorizationEndDate",
+    render: (text) => <div title="Authorization End Date">{text}</div>,
+  },
+
+  {
+    title: "First Issued",
+    dataIndex: "firstIssued",
+    key: "firstIssued",
+    render: (text) => <div title="First Issued">{text}</div>,
+  },
+  {
+    title: "Last Amended",
+    dataIndex: "lastAmended",
+    key: "lastAmended",
+    render: (text) => <div title="Last Amended">{text}</div>,
+  },
+  {
+    title: "",
+    dataIndex: "addEditButton",
+    key: "addEditButton",
+    align: "right",
+    render: (text, record) => {
+      const menu = (
+        <Menu>
+          <Menu.Item key="0">
+            <button
+              type="button"
+              className="full add-permit-dropdown-button"
+              onClick={(event) =>
+                record.openAddAmalgamatedPermitModal(event, text.guid, text.permit_no)
+              }
+            >
+              <div>
+                <Icon
+                  type="plus"
+                  className="padding-small add-permit-dropdown-button-icon"
+                  theme="outlined"
+                />
+                {text.hasAmalgamated ? "Add amalgamated amendment" : "Add amalgamated permit"}
+              </div>
+            </button>
+          </Menu.Item>
+          {!text.hasAmalgamated && (
+            <Menu.Item key="1">
+              <button
+                type="button"
+                className="full add-permit-dropdown-button"
+                onClick={(event) =>
+                  record.openAddPermitAmendmentModal(event, text.guid, text.permit_no)
+                }
+              >
+                <div>
+                  <Icon
+                    type="plus"
+                    className="padding-small add-permit-dropdown-button-icon"
+                    theme="outlined"
+                  />
+                  Add permit amendment
+                </div>
+              </button>
+            </Menu.Item>
+          )}
+          <Menu.Item key="2">
+            <button
+              type="button"
+              className="full"
+              onClick={(event) =>
+                record.openEditPermitModal(event, text.guid, text.permit_no, record.description)
+              }
+            >
+              <img alt="document" className="padding-small" src={BRAND_PENCIL} />
+              &nbsp;&nbsp;&nbsp;Edit permit status
+            </button>
+          </Menu.Item>
+        </Menu>
+      );
+      return (
+        <AuthorizationWrapper inTesting>
+          <AuthorizationWrapper permission={Permission.CREATE} isMajorMine={text.major_mine_ind}>
+            <Dropdown className="full-height full-mobile" overlay={menu} placement="bottomLeft">
+              <Button type="secondary" className="permit-table-button">
+                <div className="padding-small">
+                  <img className="padding-small--right icon-svg-filter" src={EDIT} alt="Add/Edit" />
+                  Add/Edit&nbsp;&nbsp;
```

Comment:
There are a number of non-breaking spaces in this file. I could be wrong, but I don't think we've needed them much elsewhere. What's the use-case here?

Reply:
Looks only to be padding. Updated to a style.